Sources/Classes/signature/Client/KeyModel.swift

```diff
@@ -9,9 +9,11 @@ internal struct KeyModel: Codable, Equatable {
         case pem = "pemKey"
     }
 
+    // swiftlint:disable unneeded_synthesized_initializer
```

Comment:
Sorry, I did not see this and I have added this rule to SwiftLint.yaml file in my PR#504. Please let me know before merging my PR if I need to revert it.